src/util/CMakeLists.txt

```diff
@@ -56,8 +56,10 @@ libcvc4_add_sources(
   sexpr.h
   smt2_quote_string.cpp
   smt2_quote_string.h
-  statistics.cpp
-  statistics.h
+  statistics_public.cpp
+  statistics_public.h
+  statistics_reg.cpp
+  statistics_reg.h
```

Comment:
Will these files be renamed when we remove the old statistics registry?

Reply:
Yes.